great-idea-website/css/index.css

```diff
@@ -63,4 +63,168 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+
+.pg-container {
+	width: 880px;
+	margin: auto;
+	position: relative;
+}
+
+.top-header {
+	width: 650px;
+	display: inline-block;
+	position: fixed;
+	background-image: linear-gradient(to left, rgb(25, 37, 148), black);
+	z-index: 1;
+	height: 40px;
+} 
+
+.top-header nav {
+	font-size: 16px;
+	margin-top: 10px;
+}
+
+.top-header nav a {
+	padding-right: 60px;
+	text-decoration: none;
+	color: lightgray;
+}
+
+.top-header nav a:first-child {
+	margin-left: 15px;
+}
+
+.top-header nav a:last-child {
+	padding-right: 0
+}
+
+.top-header img {
+	position: absolute;
+	right: -230px;
+	top: 0;
+}
```

Comment:
Consider wrapping that Great Idea! img with another div, and put a white background on that div. That way, when you scroll down the page the image won't blend into the content